src/main/java/com/paskar/email/application/controller/MainController.java

```diff
@@ -0,0 +1,84 @@
+package com.paskar.email.application.controller;
+
+import com.paskar.email.application.model.Email;
+import com.paskar.email.application.service.EmailRepositoryForHibernateIml;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.time.LocalDate;
+
+@Controller
+public class MainController {
+
+    private final EmailRepositoryForHibernateIml emailRepository;
```

Comment:
For this task you should have been injected EmailRepository (not actual implementation) to show that your code does not change except underlying implementation.